pychromecast/socket_client.py

```diff
@@ -215,6 +214,11 @@ def __init__(
         self.connecting = True
         self.first_connection = True
         self.socket: socket.socket | ssl.SSLSocket | None = None
+        self.selector = selectors.DefaultSelector()
+        self.self_selector_key = self.selector.register(
+            self.socketpair[0], selectors.EVENT_READ
+        )
+        self.socket_selector_key: selectors.SelectorKey | None = None
```

Comment:
Can we try to come up with more descriptive names? The socketpair is used to wake up the thread, the socket is used to talk to a remote cast device.

Reply:
`self_selector_key` -> `wakeup_selector_key`
`socket_selector_key` -> `remote_selector_key`

Does that make more sense ?

Reply:
yeah, that's more clear I think